docker-compose.yaml

```diff
@@ -1,10 +1,10 @@
 version: "3"
 services:
   oauth:
-    image: mojdigitalstudio/nomis-oauth2-server:latest
+  image: mojdigitalstudio/nomis-oauth2-server:latest
```

Comment:
Is this right?

Reply:
Just automatic formatting done by my IDE. I was messing around with it during my testing, removing bits etc. The file is still valid though.